src/components/Toast/utils/getToastTransitionStyles/getToastTransitionStyles.ts

```diff
@@ -0,0 +1,42 @@
+import { CSSProperties } from "react"
+import { TransitionStatus } from "react-transition-group"
+import { Position } from "../../types"
+
+interface Args {
+  animationDuration: number
+  transitionStatus: TransitionStatus
+  position: Position
+  behavior: "replace" | "stack"
+}
+
+/**
+ * getToastTransitionStyles
+ * ------------------------------------------------
+ * This util is used to generate a transition styles object
+ * based on animation duration, transition state, position,
+ * and behavior.
+ */
+export const getToastTransitionStyles = ({
+  animationDuration,
+  transitionStatus,
+}: Args) => {
+  const defaultStyle: CSSProperties = {
+    transition: `transform ${animationDuration}ms ease-in-out`,
+    width: 300,
```

Comment:
Yeah, that makes a lot of sense. I think I was just moving through so many iterations and refactors with all of this, that some weird artifacts were born from that. I'll just move that over to the actual Toast stylesheet and it will be more clear where to change those properties.